.github/workflows/test.yml

```diff
@@ -1,12 +1,16 @@
-name: Testing
+name: Testing Go Installer
 
 on:
   push:
+    branches: [main]
   pull_request:
 
 jobs:
-  test-go-installer:
-    runs-on: ubuntu-latest
+  os_matrix:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

Comment:
Thanks a lot for your PR. 🚀 

Unfortunately, the script is not designed to work on Windows so please remove `windows-latest` from os matrix.
<img width="585" alt="Screenshot 2023-11-08 at 9 37 38 AM" src="https://github.com/kerolloz/go-installer/assets/36763164/397507cb-5ecb-4fb1-9d10-81851e375757">
As you can see the test.sh file didn't run on windows..
```suggestion
        os: [ubuntu-latest, macos-latest]
```